From khali@linux-fr.org Sun Feb  5 14:16:49 2006
Date: Sun, 5 Feb 2006 23:17:34 +0100
From: Jean Delvare <khali@linux-fr.org>
To: Greg KH <greg@kroah.com>
Subject: [PATCH 03/11] w83627hf: Document the reset module parameter
Message-Id: <20060205231734.3ec8f2e6.khali@linux-fr.org>
Content-Disposition: inline; filename=hwmon-w83627hf-document-reset-param.patch

Document the reset module parameter which was recently added to the
w83627hf driver.

Signed-off-by: Jean Delvare <khali@linux-fr.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 Documentation/hwmon/w83627hf |    4 ++++
 1 file changed, 4 insertions(+)

--- gregkh-2.6.orig/Documentation/hwmon/w83627hf
+++ gregkh-2.6/Documentation/hwmon/w83627hf
@@ -36,6 +36,10 @@ Module Parameters
   (default is 1)
   Use 'init=0' to bypass initializing the chip.
   Try this if your computer crashes when you load the module.
+* reset: int
+  (default is 0)
+  The driver used to reset the chip on load, but does no more. Use
+  'reset=1' to restore the old behavior. Report if you need to do this.
 
 Description
 -----------
